lib/src/widgets/chat/ds_image_message_bubble.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:blip_ds/src/controllers/chat/ds_image_message_bubble.controller.dart';
+import 'package:blip_ds/src/widgets/ds_cached_network_image_view.widget.dart';
```

Comment:
utils/ is missing from path. Change to package:blip_ds/src/widgets/utils/ds_cached_network_image_view.widget.dart

Reply:
I removed, is unnecessary because now, all of the used elements are also provided by the import of 'package:blip_ds/blip_ds.dart'.